src/Pages/HumansAtNasa/HumansAtNasa.tsx

```diff
@@ -0,0 +1,18 @@
+﻿﻿import React from "react";
+import {NavBar} from "../../NavBar/NavBar";
+import {HumanAtNasa} from "../../Components/HumanAtNasa/HumanAtNasa";
+import './HumansAtNasa.scss';
+
+export function HumansAtNasa() {
+    return (
+        <div className='humansAtNasaBody'>
+            <NavBar/>
+            <h1>Humans At Nasa</h1>
+            <p>NASA's Mars missions are designed, built and operated by some of the best and brightest. Meet some of the amazing people who do their best work on another planet.</p>
+            <HumanAtNasa name={"Laura Kerber"} jobTitle={"Research Scientist"} jobDescription={"Laura Kerber is a research scientist at JPL with interests in physical volcanology, aeolian geomorphology, wind over complex surfaces, the ancient Martian climate."} imageUrl={"https://science.jpl.nasa.gov/images/people-3979.jpg"}/>
+            <HumanAtNasa name={"Dr. Timothy Parker"} jobTitle={"Geology Team Group Lead"} jobDescription={"Dr. Tim Parker was the originator of the Mars Ocean hypothesis. He earned his Ph.D. at the University of Southern California in 1994, with a dissertation entitled \"Martian Paleolakes and Oceans.\""} imageUrl={"https://science.jpl.nasa.gov/images/people-135.jpg"}/>
+            <HumanAtNasa name={"Nathan Williams"} jobTitle={"Planetary Geologist"} jobDescription={"Nathan Williams is a planetary geologist with particular interest in the remote sensing of planetary surfaces, especially relating to structural geology, geodesy, and landform evolution."} imageUrl={"https://science.jpl.nasa.gov/images/people-4770.jpg"}/>
```

Comment:
This works totally fine, but would be a little bit of a pain to add new people etc.

We _could_ make this a little bit more future proof by splitting out the data and the display. ie

```js
const nasaEmployees = [
    { 
        name: "Laura Kerber",
        jobTitle: "Research Scientits",
        jobDescription: "Laura Kerber is a research scientist at JPL with interests in..."
    },
    ...
];
```

and then in the component we can just have a loop
```jsx
{nasaEmployees.map(person => 
    <HumanAtNasa name={person.name} jobTitle={person.jobTitle} jobDescription={person.jobDescription}/>
)}
```

It probably isn't worth changing anything now (yours looks fine) just wanted to show you some options! :)

Reply:
ah yes this is a neat map method!